### DISCUSSION TITLE
DrawableRoom sometimes shows wrong background on currently playing item in lobby listing

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16113] [Q&A] [answered]
https://user-images.githubusercontent.com/53081750/146358594-aa330456-8dd6-4700-acf4-261e9c3997a7.mp4

[database.log](https://github.com/ppy/osu/files/7726616/database.log)
[network.log](https://github.com/ppy/osu/files/7726618/network.log)
[performance.log](https://github.com/ppy/osu/files/7726619/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7726620/runtime.log)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16115
